chart/values.yaml

```
@@ -1,5 +1,5 @@
 # Default values for suricata.
-replicaCount: 1
+namespace: suricata
```

```suggestion
namespace: suricata
```
We'll need to remove this if we go with the `Release.Namespace` from above.